### PR TITLE
Add name & labels to container portMapping

### PR DESCRIPTION
**Pull request by Mongey**

### CODE REVIEW

File: docker.go [hunk 1]
```diff
@@ -30,10 +30,12 @@ type Container struct {
 
 // PortMapping is the portmapping structure between container and mesos
 type PortMapping struct {
-	ContainerPort int    `json:"containerPort,omitempty"`
-	HostPort      int    `json:"hostPort"`
-	ServicePort   int    `json:"servicePort,omitempty"`
-	Protocol      string `json:"protocol,omitempty"`
+	ContainerPort int                `json:"containerPort,omitempty"`
+	HostPort      int                `json:"hostPort"`
+	Labels        *map[string]string `json:"labels,omitempty"`
```
Comment (timoreimann) [inline docker.go]: We normally provide convenience methods to easily populate pointerized variables. For labels, we already have something to [add](https://github.com/gambol99/go-marathon/blob/63fea16e82279c1d934bcb4602614308b69f22c3/application.go#L238) and [clear labels](https://github.com/gambol99/go-marathon/blob/63fea16e82279c1d934bcb4602614308b69f22c3/application.go#L250). I think we should duplicate these functions on `PortMapping`.

(I initially thought we could provide a single implementation on a custom `map[string]string` type but came to believe this won't work due to the fact that we can't initialize a receiver map on a receiver method.)

**Comment by timoreimann** [comment]
@Mongey thanks for the PR. Left one remark.

**Comment by timoreimann** [comment]
I noticed something else: We should also update [ExposePort](https://github.com/Mongey/go-marathon/blob/6f861d3325a02d32170f5c2c393acaeab72c683e/docker.go#L162) so that the new port mapping values can be easily attached to a `Docker` instance. My suggestion would be to replace the four individual parameters by a single `PortMapping` instance. As an additional benefit, callers will be able to pass in just the `PortMapping` fields they are interested in.

`Expose` and `ExposeUDP` will have to be updated along the way. This should be trivial, though.

**Comment by timoreimann** [comment]
Could you please remove (i.e., git amend and force-push) the `Fixes #190` part from commit 8fcf971 to prevent multiple references on the related issue, especially as we continue to work on this PR?

It's usually better to reference issues via the pull request description.

**Comment by Mongey** [comment]
@timoreimann thanks for the feedback. Pushed up some changes

**Comment by timoreimann** [comment]
@Mongey Would you mind adding a quick test for the new `PortMapping` label convenience methods? I think we can pretty much duplicate [TestApplicationLabels](https://github.com/gambol99/go-marathon/blob/0d84001b1247ffd4295c644b986db075395d1a3e/application_test.go#L137). (Ideally, I'd like one test to cover both label types; not sure though how to achieve that elegantly given that the convenience methods are defined on different types.)

After that, I think we can merge your PR. 👍

**Comment by timoreimann** [comment]
Great, LGTM.

Thanks again, @Mongey. 👏

**Comment by Mongey** [comment]
🙇
